File: mario.c
#include <stdio.h>
#include <cs50.h>

void prchars(int n, char c);

int main(void)
{

    int height = 24;

    while (height > 23 || height < 0)
    {
        printf("Pyramid height: ");
        height = get_int();
    }

    int ihash = 1;
    int ispace = height - 1;

    for (int i = 0; i < height; i++)
    {
        prchars(ispace, ' ');
        prchars(ihash, '#');

        printf("  ");
        prchars(ihash, '#');

        printf("\n");

        ispace--;
        ihash++;
    }


}

void prchars(int n, char c)
{
    for (int i = 0; i < n; i++)
    {
        printf("%c", c);
    }
}

File: mario.py
import cs50

def main():
    #get pyramid height
    while True:
        print("Pyramid height:", end="")
        height = cs50.get_int()
        if height < 23 and height > 0:
            break

    #set initial variables
    ihash = 1
    ispace = height - 1

    #make pyramids!
    for i in range(height):
        prchars(ispace, " ")
        prchars(ihash, "#")

        print("  ", end="")

        prchars(ihash, "#")
        print("")

        ispace -= 1
        ihash += 1

def prchars(n, c):
    for i in range(n):
        print(c, end="")

if __name__ == "__main__":
    main()